### PR TITLE
Correct hadoop version in RELEASE file

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

correct verion of

- Spark Hadoop
- Kyuubi Hadoop

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: bin/load-kyuubi-env.sh [hunk 1]
```diff
@@ -72,8 +72,8 @@ export KYUUBI_SCALA_VERSION="${KYUUBI_SCALA_VERSION:-"2.12"}"
 
 if [[ -f ${KYUUBI_HOME}/RELEASE ]]; then
   KYUUBI_VERSION="$(grep "Kyuubi " "$KYUUBI_HOME/RELEASE" | awk -F ' ' '{print $2}')"
-  SPARK_VERSION_BUILD="$(grep "Spark " "$KYUUBI_HOME/RELEASE" | awk -F ' ' '{print $2}')"
-  HADOOP_VERSION_BUILD="$(grep "Hadoop " "$KYUUBI_HOME/RELEASE" | awk -F ' ' '{print $2}')"
+  SPARK_VERSION_BUILD="$(grep "Spark " "$KYUUBI_HOME/RELEASE" | awk -F ' ' '{print $2}' | head -1)"
```
Comment (yaooqinn) [inline bin/load-kyuubi-env.sh:75]: grep -v？

File: build/dist [hunk 2]
```diff
@@ -184,7 +184,8 @@ echo "Kyuubi $VERSION $GITREVSTRING built for" > "$DISTDIR/RELEASE"
 echo "Java $JAVA_VERSION" >> "$DISTDIR/RELEASE"
 echo "Scala $SCALA_VERSION" >> "$DISTDIR/RELEASE"
 echo "Spark $SPARK_VERSION" >> "$DISTDIR/RELEASE"
-echo "Hadoop $SPARK_HADOOP_VERSION" >> "$DISTDIR/RELEASE"
+echo "Spark Hadoop $SPARK_HADOOP_VERSION" >> "$DISTDIR/RELEASE"
```
Comment (yaooqinn) [inline build/dist:187]: do we rely on spark-hadoop to build for the compiler?

Comment (ulysses-you) [reply build/dist:187]: yes, it actullay affect the built-in spark directory name, like `spark-3.1.2-hadoop2.7` the version is 2.7

Comment (yaooqinn) [reply build/dist:187]: I mean the compiler, we don't rely on any hadoop jar from spark dist for compiling, right?

Comment (ulysses-you) [reply build/dist:187]: that's right, it only used to help find the built-in SPARK_HOME

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**